_posts/2020-11-07-brief-history-of-programming-languages.md

```diff
@@ -0,0 +1,103 @@
+Maurice Wilkes and his team at University of Cambridge constructed one of the first electronic digital stored-program computer - the The Electronic delay storage automatic calculator in 1949. The machine was capable of adding, subtracting, multiplying-and-adding, AND-and-adding, shifting left, arithmetic right shifting, printing characters and a few more but no division was possible. The instructions were held in a 18 bit format, the topmost bit could not be used due to timing issues, therefore only 17 bits were available. Assembly language is any low level programming language which is specifically designed for a particular computer architecture. It simplifies the language of machine code and the instructions are needed to tell the computer what to do. On the contrary, high level programming languages are portable across multiple systems.
+Maurice Wilkes and his team at University of Cambridge constructed one of the first electronic digital stored-program computer - the The Electronic delay storage automatic calculator in 1949. The machine was capable of adding, subtracting, multiplying-and-adding, AND-and-adding, shifting left, arithmetic right shifting, printing characters and a few more but no division was possible. The instructions were held in a 18 bit format, the topmost bit could not be used due to timing issues, therefore only 17 bits were available.   
+
+![](https://upload.wikimedia.org/wikipedia/commons/2/22/EDSAC_%2819%29.jpg)Electronic delay storage automatic calculator (EDSAC)
+
+## Autocode
+
+Alick Glennie develped Autocode at the University of Manchester in the early 1950s, which is believed to be the first compiled programming language, which means that it can be translated directly into machine code using complier - a program primarily used for translation of high-level language to a lower level in order to create an executable program.
+
+## Fortran
+
+In 1954 Fortran (which stands for Formula Translation) was created by John Backus. It was the first widely used general purpose programming language that was used for complication mathematical, scientific and statistical work. When introduced, it was met with skepticism because of delays in development and bugs but eventually became known for its efficiency. It is one of the oldest languages still used today for programs that benchmark the world's fastest computers.
+
+## AGOL and COBOL
+
+Created in 1958 and 1959 respecively. Agol stands for Algorithmic Language and was created for scientific use. It is a predecessor of Pascal, C, C++, and Java. COBOL stands for COmmon Business Oriented Language and was created by  Dr. Grace Murray Hopper. It was designed for business and it is still used in ATMs, credit card processing, telephone systems, hospital and government computers, automotive systems, and traffic signals. *Fun fact*: Pieces of COBOL source code were used in Terminator's vision display in the movie "The Terminator".
```

Comment:
Grammar: typo - "C0mmon Business Oriented Language"; extra space after "by" in "was created by Dr. Grace Murray Hopper."

Reply:
CO is intentionally capitalized to show where the abbreviation COBOL came from. Thanks for the double space remark!